### ISSUE TITLE
Imgix, crop & sizing with Gatsby

### ISSUE THREAD

**Issue by Zellement** [angeloashmore/gatsby-source-prismic#206]
Hello!

Dreadfully sorry - I'm new to using Imgix and since upgrading I can't get my head around what I'm doing wrong, so I hope someone can help me or point me in the right direction.

Since upgrading to 3.0.0 by following [this guide](https://github.com/angeloashmore/gatsby-source-prismic/blob/master/docs/migrating-from-v2-to-v3.md) some of my images now appear oddly cropped, and I can't seem to get them to behave in the CMS.

In the CMS, I have [cropped an image to be 1000x825px](http://prntscr.com/rmhfwm). 

I have the following graphql code to return the images;

```
... on PrismicPagePageContentTextGallery {
            id
            items {
              image {
                fluid(maxWidth:1000,maxHeight:825) {
                    ...GatsbyPrismicImageFluid
                }
              }
            }
          }
```

But when the image returns, it's just the [full sized one](https://images.prismic.io/formula1gym/6ec6c758-d868-4571-9ecb-9cce7f2b7a95_MVIMG_20200217_170908.jpg?auto=compress%2Cformat&fit=max&w=2000&h=1650)

Am I completely missing something here, haha? It doesn't seem to be listening or adhering to the size I am cropping inside the CMS since the switch. It worked beautifully beforehand when it was using localFile?

Any help or suggestion would be very welcome.

Thanks.

**Comment by angeloashmore** [comment]
Ah, it looks like the cropping is not being sent to the URLs. We're using [`ts-imgix`](https://github.com/unsplash/ts-imgix) to generate the Imgix URLs, but I don't think it supports `rect`, unfortunately.

I'll see if I can get a PR going there to add support (or you can if you're up to it!).

For now, you can either replace the image with an already cropped version, or stick to `localFile` until this is fixed. Sorry for the trouble!

**Comment by Zellement** [comment]
Ah, no worries, thanks for the update @angeloashmore 

Glad I wasn't going insane.

**Comment by angeloashmore** [comment]
@Zellement This should be resolved now in the just published `v3.1`. Imgix params are preserved from Prismic, including the `rect` option created by cropping in Prismic's UI.

The new version also supports any Imgix URL parameter using the `imgixParam` argument on the image field if you need more control.
